schemas/dns.yml

```
@@ -205,6 +205,6 @@
         data formats it can contain. Extracting all IP addresses seen in there to
         `dns.resolved_ip` makes it possible to index them as IP addresses, and
         makes them easier to visualize and query for.
-      example: [10.10.10.10, 10.10.10.11]
+      example: "[10.10.10.10, 10.10.10.11]"
```

In the previous example, YAML was complementing my laziness and turning each array element into a string automatically.

Now that the string is quoted, we should make sure to double quote each element within, so that the example in the docs looks like a valid JSON snippet.

```suggestion
      example: '["10.10.10.10", "10.10.10.11"]'
```